src/libraries/System.Text.Json/ref/System.Text.Json.csproj

```diff
@@ -7,6 +7,7 @@
 
   <ItemGroup>
     <Compile Include="System.Text.Json.cs" />
+    <Compile Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'" Include="System.Text.Json.Typeforwards.netcoreapp.cs" />
```

Comment:
(nit) It is a little surprising to put this condition inline on the `Compile`. The other similar conditions are all on the ItemGroup, so you can group them together.

What do you think about pulling this out into a new ItemGroup just for .NETCoreApp Compile items?

```xml
  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
    <Compile Include="System.Text.Json.Typeforwards.netcoreapp.cs" />
  </ItemGroup>
```

Reply:
I did this because it is a one-off and I didn't foresee others.  I'm fine changing to the more verbose syntax.